README.md

```diff
@@ -4,3 +4,255 @@ lua-client
 [![Build Status](https://travis-ci.org/neovim/lua-client.svg?branch=master)](https://travis-ci.org/neovim/lua-client)
 
 Lua client for Neovim
+
+
+Basic usage
+-----------
+
+The module `nvim.nvim` provides a set of functions to connect to an instance of neovim.
+
+### Create an instance of the client
+
+First of all we need to get an instance of the client ready to talk with neovim.
+
+#### new_from_socket_file
+
+Connects to a running neovim instance using a socket file stream.
+
+```lua
+local Nvim = require('nvim.nvim')
+local nvim = Nvim.new_from_socket_file('/tmp/.nvim.socket')
+```
+
+#### new_from_socket
+
+Connects to a running neovim instance using a socket tcp stream.
+
+```lua
+local Nvim = require('nvim.nvim')
+local nvim = Nvim.new_from_socket('127.0.0.1', 6666)
+```
+
+#### new_from_process
+
+```lua
+local Nvim = require('nvim.nvim')
+local nvim = Nvim.new_from_process()
+```
+
+This code will create a new embedded neovim instance and will connect to it.
+
+All the functions creating a new Nvim instace accept a version parameter (numeber) which
+specifies what version of the API to use. The only valid values for that version number are 1 and 0.
+
+By default API version 1 is used, that means that functions / methods deprecated in that API version
```

Comment:
Here you use a space between the slash, but further down you do not. Stay consistent.